aspnetcore/fundamentals/error-handling.md

```diff
@@ -77,70 +57,57 @@ public IActionResult Error()
 
-::: moniker-end
-
-::: moniker range="= aspnetcore-2.0"
-
-The middleware is made available by the [Microsoft.AspNetCore.Diagnostics](https://www.nuget.org/packages/Microsoft.AspNetCore.Diagnostics/) package, which is available in the [Microsoft.AspNetCore.All metapackage](xref:fundamentals/metapackage).
-
-::: moniker-end
-
-::: moniker range="< aspnetcore-2.0"
-
-The middleware is made available by adding a package reference for the [Microsoft.AspNetCore.Diagnostics](https://www.nuget.org/packages/Microsoft.AspNetCore.Diagnostics/) package in the project file.
-
-::: moniker-end
-
 Add a line to the `Startup.Configure` method:
 
 ```csharp
 app.UseStatusCodePages();
 ```
 
-<xref:Microsoft.AspNetCore.Builder.StatusCodePagesExtensions.UseStatusCodePages*> should be called before request handling middlewares in the pipeline (for example, Static File Middleware and MVC Middleware).
+Call the <xref:Microsoft.AspNetCore.Builder.StatusCodePagesExtensions.UseStatusCodePages*> method before request handling middleware (for example, Static File Middleware and MVC Middleware).
 
-By default, Status Code Pages Middleware adds text-only handlers for common status codes, such as 404:
+By default, Status Code Pages Middleware adds text-only handlers for common status codes, such as *404 - Not Found*:
 
-![404 page](error-handling/_static/default-404-status-code.png)
+```
+Status Code: 404; Not Found
+```
 
 The middleware supports several extension methods. One method takes a lambda expression:
 
 [!code-csharp[](error-handling/samples/2.x/ErrorHandlingSample/Startup.cs?name=snippet_StatusCodePages)]
 
-An overload of `UseStatusCodePages` takes a content type and format string:
+An overload of <xref:Microsoft.AspNetCore.Builder.StatusCodePagesExtensions.UseStatusCodePages*> takes a content type and format string:
 
 ```csharp
 app.UseStatusCodePages("text/plain", "Status code page, status code: {0}");
 ```
+
 ### Redirect re-execute extension methods
 
 <xref:Microsoft.AspNetCore.Builder.StatusCodePagesExtensions.UseStatusCodePagesWithRedirects*>:
 
 * Sends a *302 - Found* status code to the client.
-* Redirects the client to the location provided in the URL template. 
+* Redirects the client to the location provided in the URL template.
 
-The template may include a `{0}` placeholder for the status code. The template must start with a forward slash (`/`).
+The template may include a placeholder (`{0}`) for the status code. The template must start with a forward slash (`/`).
 
 [!code-csharp[](error-handling/samples/2.x/ErrorHandlingSample/Startup.cs?name=snippet_StatusCodePagesWithRedirect)]
 
 <xref:Microsoft.AspNetCore.Builder.StatusCodePagesExtensions.UseStatusCodePagesWithReExecute*>:
 
 * Returns the original status code to the client.
-* Specifies that the response body should be generated by re-executing the request pipeline using an alternate path. 
+* Generates the response body by re-executing the request pipeline using an alternate path.
```

Comment:
Can we explain better the difference between redirect and reexecute?  What scenarios call for reexecute?